src/Reader/Reader.php

```diff
@@ -235,7 +241,7 @@ public static function import($uri, $etag = null, $lastModified = null)
             if ($data) {
                 return static::importString($data);
             }
-            $response = $client->send();
+            $response = $client->get($uri);
```

Comment:
I've removed a couple of notices but this one still aplicable https://github.com/zendframework/zend-feed/pull/14#discussion_r45259870